server/classes.py

```diff
@@ -265,6 +255,43 @@ def setName(self, name):
             raise Exception("The name of the Calisthenics workout is too long")
 
 
+class workoutPlan(ABC):
```

Comment:
Does this add another workoutPlan Class? I'm kinda confused, unless this is just moving the code idk?